spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -24,28 +24,101 @@
 
   let(:public_key_credential) { client.create(challenge: original_challenge) }
 
-  before do
-    WebAuthn.configuration.origin = origin
+  shared_context "with a single origin" do
+    before do
+      WebAuthn.configuration.origin = origin
+    end
   end
 
-  context "when everything's in place" do
+  shared_context "with multiple origins" do
+    let(:allowed_origins) do
+      [
+        fake_origin,
+        "android:apk-key-hash:blablablablablalblalla"
+      ]
+    end
+
+    before do
+      WebAuthn.configuration.origin = allowed_origins
+    end
+  end
+
+  shared_context "with rp_id set to" do |rp_id|
+    before do
+      WebAuthn.configuration.rp_id = rp_id
+    end
+  end
+
+  shared_examples "is valid and verifies" do
```

Comment:
What do you think about changing the name of this shared example to `a valid attestation response` so when included it reads like:

```
it_behaves_like 'a valid attestation response'
```

Reply:
sure, makes sense to me. Will switch to that